casanovo/denovo/model_runner.py

```diff
@@ -116,36 +116,25 @@ def train(
             self.loaders.val_dataloader(),
         )
 
-    def evaluate(self, peak_path: Iterable[str]) -> None:
-        """Evaluate peptide sequence preditions from a trained Casanovo model.
-
-        Parameters
-        ----------
-        peak_path : iterable of str
-            The path with MS data files for predicting peptide sequences.
-
-        Returns
-        -------
-        self
-        """
-        self.initialize_trainer(train=False)
-        self.initialize_model(train=False)
-
-        test_index = self._get_index(peak_path, True, "evaluation")
-        self.initialize_data_module(test_index=test_index)
-        self.loaders.setup(stage="test", annotated=True)
-
-        self.trainer.validate(self.model, self.loaders.test_dataloader())
-
-    def predict(self, peak_path: Iterable[str], output: str) -> None:
+    def predict(
```

Comment:
**question**: Is it alright to have all of the model output and spectrum annotations in memory at once or does this need to be done in batches?